wiki/Reporting_Bad_Behaviour/en.md

```diff
@@ -1,133 +1,112 @@
+Parameters in **<** and **>** are optional.
 
-Take note that <b>&lt;</b> and <b>&gt;</b> are not necessary.
+If the user has a space at their username, it must be  replaced with an underscore ``_`` because the osu! ingame chat is based on [IRC](IRC wiki/Internet_Relay_Chat).
 
-If the user has a space at their username, replace it with an underscore ('\_') because our chat is based on [IRC](IRC "wikilink").
+``This User is spamming in #channel`` turns to ``This_User is spamming in #channel``
 
-`User: This User`
-`Example: This_User is spamming in #channel`
+Reports are not case-sensitive so it is okay to use lowercase letters like ``this_user``.
 
-Reports are not case-sensitive so it's okay to use lowercase letters like **this\_user**.
+![Example of using the !report command](img/Report-command.png "Example of using the !report command")
 
-<center>
-<img src="Report-command.png" title="Example of using the !report command" alt="Example of using the !report command" width="400" />
 
-</center>
 ### Using the report button ingame
 
-`Report the user by this way only needs a type of reason and your custom text, unlike the !report command.`
+Reporting the user by this way only needs a type of reason and the custom text, unlike the `!report` command.
+
+![Click the report button](img/Report-user-1.png "Click the report button")
+
+![Remember to add a reason here!](img/Report-user-2.png "Remember to add a reason here!")
 
-<center>
-<img src="Report-user-1.png" title="fig:Click the report button" alt="Click the report button" width="400" /> <img src="Report-user-2.png" title="fig:Remember to add a reason here!" alt="Remember to add a reason here!" width="400" />
 
-</center>
-### Using the report button at the user's profile
+### Using the report button in the profile of a user
 
 This also works the same as the report button ingame.
 
 Remember to add details to your report (like translations) at the comments.
 
-<center>
-<img src="Report-user-profile.png" title="Report-user-profile.png" alt="Report-user-profile.png" width="400" />
+![This will open a dialog for reporting the user.](img/Report-user-profile.png "This will open a dialog for reporting the user.")
 
-</center>
-### Using the report link at the user's record at beatmap listing
 
-Only use this when you are reporting users for cheating.
+### Using the report link at the record of a user at beatmap listing
 
-<center>
-<img src="Report-user-beatmap.png" title="Report-user-beatmap.png" alt="Report-user-beatmap.png" width="400" />
+This should only be used when cheating users are reported.
+
+![Pressing this link will report that score.](img/Report-user-beatmap.png "Pressing this link will report that score.")
 
-</center>
 
-------------------------------------------------------------------------
 
 Examples what to report
 -----------------------
 
-:\* **Spamming or flooding in chat.**
-
-Using a broken keyboard is not an excuse!
+* **Spamming or flooding in chat.**
```

Comment:
Style suggestion (you do not have to follow):-

Rather than using the current bullet points, how about using level 3 headers? It will provide a nice link in the osu-web's TOC that the user can press on rather than searching for it.


For example for this line:-

Replace
`* **Spamming or flooding in chat.**`
To
`### Spamming or flooding in chat`